### PR TITLE
Feature/2020/dec05

### PR DESCRIPTION
**Pull request by ablqk**

### CODE REVIEW

File: 2020/dec04/dec04.go [hunk 1]
```diff
@@ -23,6 +23,7 @@ type dec04 struct {
 	input string
 }
```
Comment (ablqk) [inline 2020/dec04/dec04.go:25]: First commit has a misleading number.

File: 2020/dec05/dec05.go [hunk 2]
```diff
@@ -13,5 +18,34 @@ type dec05 struct {
...
+	})
+	if err != nil {
+		return "", err
+	}
+	length++
+	// there is probably a problem of luck depending on whether the missing seat is after or before the middle of the plane
+	// we compute the expected sum, were all seats actually figuring in the boarding passes
+	expectedSum := int64((highest.SeatID()+lowest.SeatID())*length) / 2
+	// then we substract the actual sum and return the difference
+	return fmt.Sprintf("The highest seat ID is %d", expectedSum-sum), nil
```
Comment (floppyzedolfin) [inline 2020/dec05/dec05.go:50]: that was smart :)

Comment (ablqk) [reply 2020/dec05/dec05.go:50]: Not my idea. I'm bad at inventing algorithms, and rather good at googling them.

**Pull request merged**